src/inews-mixins/playlist.ts

```diff
@@ -42,7 +42,7 @@ export function getRundownWithBackTime(
 	manifest: BlueprintResultRundown
 ): BlueprintResultRundown {
 	const sortedSegments = ingestRundown.segments.sort((a, b) => a.rank - b.rank)
-	const firstContinuityStory = sortedSegments.find(segment => segment.name.match(/^\s*continuity\s*$/i))
+	const firstContinuityStory = sortedSegments.find((segment) => segment.name.match(/^\s*continuity\s*$/i))
```

Comment:
Yeah, so that unfortunately happened after I upgraded prettier (because the previous version did not support template literals). I ran linting when already having lots of uncommitted changes and I didn't want to manually revert all those done by the linter.

Reply:
Personally I like it better without the parentheses, but it's something that I can live with.
It should be a decision made by the team.